src/WidgetIncluderModule.php

```diff
@@ -21,53 +21,64 @@ public function main(App $app)
 
     public function renderWidget(App $app, $widget_id, $options = [], $view = null)
     {
+
+        // get user context
         $user = $app->user();
-        
+
         /** @var Widget $widget */
         if (!$widget = Widget::where([
-            'id' => $widget_id
+            'id' => $widget_id,
```

Comment:
Is there a specific reason why you added the `,`? It is allowed in PHP, but I usually develop javascript and JSON and there it is either not allowed or should be avoided. So my first decision was to also don't have it here. Maybe there is a reason to do different :)